Package.swift

```diff
@@ -6,57 +6,34 @@ let package = Package(
     platforms: [.iOS(.v11), .macOS(.v10_13), .tvOS(.v11), .watchOS(.v4)],
     products: [
         .library(name: "Sentry", targets: ["Sentry"]),
-        .library(name: "Sentry-Dynamic", type: .dynamic, targets: ["Sentry"]),
-        .library(name: "SentrySwiftUI", targets: ["SentrySwiftUI"])
+        .library(name: "Sentry-Dynamic", targets: ["Sentry-Dynamic"]),
+        .library(name: "SentrySwiftUI", targets: ["Sentry", "SentrySwiftUI"])
     ],
     targets: [
-        .target(
-            name: "Sentry",
-            dependencies: ["SentryPrivate"],
-            path: "Sources",
-            sources: [
-                "Sentry/",
-                "SentryCrash/"
-            ],
-            publicHeadersPath: "Sentry/Public/",
-            cxxSettings: [
-                .define("GCC_ENABLE_CPP_EXCEPTIONS", to: "YES"),
-                .headerSearchPath("Sentry/include"),
-                .headerSearchPath("Sentry/include/HybridPublic"),
-                .headerSearchPath("Sentry/Public"),
-                .headerSearchPath("SentryCrash/Installations"),
-                .headerSearchPath("SentryCrash/Recording"),
-                .headerSearchPath("SentryCrash/Recording/Monitors"),
-                .headerSearchPath("SentryCrash/Recording/Tools"),
-                .headerSearchPath("SentryCrash/Reporting/Filters"),
-                .headerSearchPath("SentryCrash/Reporting/Filters/Tools"),
-                .headerSearchPath("SentryCrash/Reporting/Tools")
-            ]
-        ),
-        .target( name: "SentryPrivate",
-                 path: "Sources",
-                 sources: [
-                    "Swift"
-                 ],
-                 resources: [
-                    .copy("Resources/PrivacyInfo.xcprivacy")
-                 ]
-               ),
+        .binaryTarget(
+                    name: "Sentry",
+                    url: "https://github.com/getsentry/sentry-cocoa/releases/download/8.22.0-alpha.0/Sentry.xcframework.zip",
+                    checksum: "86156301aee5c8774a8cd5c240286f914f6e7721aaac5a7c9d049ea613a4b730" //Sentry-Static
+                ),
+        .binaryTarget(
+                    name: "Sentry-Dynamic",
+                    url: "https://github.com/getsentry/sentry-cocoa/releases/download/8.22.0-alpha.0/Sentry-Dynamic.xcframework.zip",
+                    checksum: "86156301aee5c8774a8cd5c240286f914f6e7721aaac5a7c9d049ea613a4b730" //Sentry-Dynamic
+                ),
```

Comment:
IMO we should bump a major and kill one of this two (remove static and change the name of the Dynamic to Sentry only)

Reply:
Yes. We are keeping two. My suggestion is for the next major